### ISSUE TITLE
UNET 3plus not learning on large datasets

### ISSUE THREAD

**Issue by jrmathew1** [yingkaisha/keras-unet-collection#41]
I am trying to use the UNET 3plus with deep supervision and cgm for nodule segmentation from lung ct scans. When I train the model on a 1000 image subset of the dataset the model learns and outputs meaningful segmentation masks. 

<img src = "https://user-images.githubusercontent.com/98428570/151202542-c3fff6dd-fd7f-4c5e-9208-9f976062c37a.PNG" width=50% height=50%>

However, when I train on a larger dataset of 2000 or more images the model does not learn and outputs a mask of all 0s always.

<img src =https://user-images.githubusercontent.com/98428570/151202555-5524890b-33c7-488f-8b5b-e6633002fbfb.PNG width=50% height=50%>

I have tried increasing and reducing the learning rate as well changing optimizers and loss functions but the issue remains. I also tried the att_unet_2d model and had the same issue. 

I have attached my code:
[training_code.zip](https://github.com/yingkaisha/keras-unet-collection/files/7943486/training_code.zip)
